plugin/storage/badger/spanstore/read_write_test.go

```diff
@@ -302,11 +302,11 @@ func TestFindNothing(t *testing.T) {
 		}
 
 		trs, err := sr.FindTraces(context.Background(), params)
-		assert.NoError(t, err)
+		assert.EqualError(t, err, "trace not found")
```

Comment:
this is incorrect, the contract for FindTraces is to return (nil, nil) when nothing is found. It's not an error to find nothing.

I will add this to the API once this PR is merged, see https://github.com/jaegertracing/jaeger/pull/2478/files#diff-a3d8538367b0e9222a46232f599e97daR37

Reply:
I see. Based on your PR, it's looks like the GetTrace() will be returning spanstore.ErrTraceNotFound. Thus, this PR seams no longer relevant.

Reply:
@bobrik would you like to close this one, in favor of #2478 ?

Reply:
@jpkrohling Yes, I think we can close it.